_data/simple-icons.json

```diff
@@ -640,6 +640,11 @@
             "hex": "0066DC",
             "source": "http://press.dailymotion.com/?page_id=346"
         },
+        {
+            "title": "Dashlane",
+            "hex": "005C7F",
```

Comment:
What do you mean by "_The color come from the website header_"? When I look at [the svg in the header](https://d38muu3h4xeqr1.cloudfront.net/website/static/WN-1589/images/header/dashlane-logo-teal.svg) I find it uses `#007C97` (not `#005C7F`) 🤔

Reply:
I mean I literally took the color from website header background., I was not sure about it. I will fix it.
![image](https://user-images.githubusercontent.com/34105974/62799300-1ab92b80-bae9-11e9-9705-7edd360d4821.png)

I used this svg file.
https://d38muu3h4xeqr1.cloudfront.net/website/static/WN-1589/images/header/dashlane-logo.svg

Reply:
Alright thanks for the clarification, I think we'll stick with the fix for now if it turns out we should use something else it can always be change later.